tsconfig.json

```diff
@@ -7,6 +7,8 @@
     "declaration": true,
     "strict": true
   },
-  "include": ["src/**/*.ts"],
-  "exclude": ["node_modules"]
+  "include": ["src/**/*.ts", "src/*.ts"],
+  "exclude": ["node_modules"],
+  "sourceRoot": "./src",
+  "sourceMap": true
```

Comment:
Production build shall not enable sourceMap. 
There's already a script to build with source map enable: `build:dev`, you just have to use it to build with sourceMap set to true.